src/main/java/uk/sky/cqlmigrate/CqlPaths.java

```diff
@@ -22,16 +27,23 @@ public CqlPaths(Map<String, Path> paths) {
     }
 
     static CqlPaths create(Collection<Path> directories) {
-        Map<String, Path> paths = new HashMap<>();
+        Map<String, Path> cqlPathsMap = new HashMap<>();
 
         directories.stream()
-                .map(path -> path.toFile().listFiles(CQL_FILE_NAMES))
-                .map(Arrays::asList)
-                .flatMap(Collection::stream)
-                .map(File::toPath)
```

Comment:
blah humans shouldn't be involved, i'll add an issue for something like checkstyle